toolkit/tools/internal/userutils/userutils.go

```diff
@@ -147,19 +148,24 @@ func UpdateUserPassword(installRoot, username, hashedPassword string) error {
 }
 
 // UserHomeDirectory returns the home directory for a user.
-func UserHomeDirectory(username string) string {
-	if username == RootUser {
-		return RootHomeDir
-	} else {
-		return filepath.Join(UserHomeDirPrefix, username)
+func UserHomeDirectory(installRoot string, username string) (string, error) {
```

Comment:
nit: `GetUserHomeDirectory`?

Reply:
`UserHomeDirectory` is the current name of the function. Also, there are those that prefer to omit the `Get` prefix for functions. (I don't mind either way.)